pgx_async/test/dune

```diff
@@ -1,15 +1,3 @@
-(executables
+(tests
```

Comment:
Does this require a particular version of dune (is dune >1.1 still accurate)?

Reply:
dune 1.1 is still dune. The tests stanza has been there since version 1.0 i believe.